src/LuaXML_101012/Makefile

```diff
@@ -1,17 +1,21 @@
 # 2009-03-16 by gf
 #
 ifeq ($(PREFIX),)
-    PREFIX := /usr
+    PREFIX ?= /usr
```

Comment:
Why we need this change?
It seems that we are already sure that PREFIX variable has an empty value:
`ifeq ($(PREFIX),)`
If you concern about whitespace characters within PREFIX
you can use the strip function:
```
ifeq ($(strip $(PREFIX)),)
    PREFIX := /usr
endif
```

Reply:
fixed